herbert_core/classes/MPIFramework/parallel_worker.m

```diff
@@ -184,8 +184,8 @@
         % something wrong with the code. We can not process interrupt
         % properly, but filebased framework should still be
         % available.
-
-        if ~strcmp(ME.identifier,'MESSAGE_FRAMEWORK:cancelled')  
+        
+        if ~strcmp(ME.identifier,'MESSAGE_FRAMEWORK:cancelled')
```

Comment:
This is ignoring all other errors. Need an `else; rethrow(ME)`

Reply:
Only if cancelled is thrown, one can continue. All other errors are dealt with later. There is so complex logic here, that `rethrow` will probably break everything.  I need to cancel job on error, but continue running worker, so no unexpected exceptions.